dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
The actual bug is in marathon. But until it's fixed, we should still catch the error.